includes/Admin/Dashboard.php

```diff
@@ -393,6 +393,8 @@ public function get_dashboard_settings(): array {
 					'users'       => '/web-stories/v1/users/',
 					'templates'   => sprintf( '/web-stories/v1/%s/', $templates_rest_base ),
 					'settings'    => '/web-stories/v1/settings/',
+					'search'      => '/wp/v2/search/',
```

Comment:
Why use search endpoint? What benefit does this give us over use just pages endpoints, like this?  
```http://localhost:8899/wp-json/wp/v2/pages?search=term```

Reply:
The search endpoint is a bit more lightweight, but I suppose I could achieve the same with the pages endpoint + using `?_fields`. I'll take a look.

Reply:
Honest question, just trying to understand the benefits here. We are already pass fields [here](https://github.com/google/web-stories-wp/pull/9051/files#diff-f2fd97f362a91bb56a0137a84ae805d30529fa2624f88d764540044248c525e5R29) already.

Reply:
Fixed now